Tests/Rules/AvoidGlobalAliases.tests.ps1

```
@@ -0,0 +1,31 @@
+﻿Import-Module PSScriptAnalyzer
+
+$AvoidGlobalAliasesError = "Avoid creating aliases with a Global scope."
+$violationName = "AvoidGlobalAliases"
+
+$directory = Split-Path -Parent $MyInvocation.MyCommand.Path
+$violations = Invoke-ScriptAnalyzer $directory\AvoidGlobalAliases.ps1 | Where-Object {$_.RuleName -eq $violationName}
+$noViolations = Invoke-ScriptAnalyzer $directory\AvoidGlobalAliasesNoViolations.ps1 | Where-Object {$_.RuleName -eq $violationName}
+
+
+Describe "$violationName " {
+    Context "When there are violations" {
+        It "Has 5 avoid using empty Catch block violations" {
```

empty catch block? Please update this description.